rosbag2_transport/src/rosbag2_transport/player.cpp

```diff
@@ -491,6 +521,18 @@ void Player::prepare_publishers()
         "Ignoring a topic '%s', reason: %s.", topic.name.c_str(), e.what());
     }
   }
+
+  if (!topic_without_support_acked.empty()) {
+    // remove the last ", "
+    topic_without_support_acked.erase(
+      topic_without_support_acked.end() - 2,
+      topic_without_support_acked.end());
+
+    RCLCPP_WARN(
+      get_logger(),
+      "--wait-for-all-acked is invaild for below topics since reliability of QOS is BestEffort.\n"
```

Comment:
Very minor change here.

```suggestion
      "--wait-for-all-acked is invaild for the below topics since reliability of QOS is BestEffort.\n"
```

Reply:
Thanks for your comments.   
Please help to check again @jhdcs